evm_arithmetization/src/witness/traces.rs

```diff
@@ -187,14 +188,14 @@ impl<T: Copy> Traces<T> {
             "generate mem_before trace",
             all_stark
                 .mem_before_stark
-                .generate_trace(mem_before_values, timing)
+                .generate_trace(mem_before_values_to_rows(mem_before_values), timing)
         );
         let mem_after_trace = timed!(
             timing,
             "generate mem_after trace",
             all_stark
                 .mem_after_stark
-                .generate_trace(&final_values, timing)
+                .generate_trace(final_values.clone(), timing)
```

Comment:
Why do we now need to clone?

Reply:
Actually, I just noticed there was a clone anyway, so nervemind my comment, sorry!